tasks/deployment/deploy-incentives-impl-v2.ts

```diff
@@ -27,23 +27,20 @@ task('deploy-incentives-impl-v2', 'Incentives controller implementation deployme
     console.log(`[StakedTokenIncentivesController] Starting deployment:`);
     const impl = await deployStakedTokenIncentivesController(
       [STAKED_STARLAY],
-      false, // TODO: revert
-      deployer
```

Comment:
cause of `initialize()` failure
https://github.com/starlay-finance/starlay-incentives-controller/pull/29#issuecomment-1035147086